.travis.yml

```diff
@@ -10,12 +10,15 @@ services:
   - docker
 before_install:
   - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
```

Comment:
If you already change the protocol of the submodule, please also remove this line